qemu/qemu.go

```diff
@@ -1180,6 +1194,12 @@ type IOThread struct {
 	ID string
 }
 
+// Incoming controls migration source preparation
+type Incoming struct {
+	// Accept incoming migration as an output from specified external command.
```

Comment:
command should start with `Exec`, isn't it?

Reply:
Yes, it is. And the prefixing `exec:` is added in `appendIncoming()`.